src/Symfony/Component/PropertyInfo/Extractor/ConstructorArgumentTypeExtractorInterface.php

```diff
@@ -17,8 +17,6 @@
  * Infers the constructor argument type.
  *
  * @author Dmitrii Poddubnyi <dpoddubny@gmail.com>
- *
- * @internal
```

Comment:
I'm not a huge fan, you did this because it's used in another component? Just ignore the error for internal use between components it's fine no?

Reply:
Yes. It is because I used it in another component. 

Do you know the reason why it is internal in the first place?